README

```diff
@@ -1,7 +1,95 @@
-Download the graphics, font, and sound files to be used in your code
-https://drive.google.com/drive/folders/1WXHI-pd8VQT9KvmAx280LxZfvcfG7jOh?usp=sharing
+"""
+Project: Python + Game
+Lead: Bernard Yap
+Team Members: Benjamin Goh, Nazrin Nazarudin, Yun Sion, Luke Leh, Lim Eejoy, Guan Zhou, Tarun, Raja Darshini
+Goal: Clone the original mario game as much as possible!
 
-Happy coding! Bernard will be your supervisor, just ask me if anything 
+Credits to justinmeister for all the resources
+https://github.com/justinmeister/Mario-Level-1/tree/master/resources/graphics
+
+click on the link below to see the real mario game
+https://supermarioplay.com/
+"""
+import pygame
+import sys
+
+#initializing pygame
+pygame.init()
+
+# create a screen instance/object with 
+# 1000px width and 448px height
+screenW = 1000
+screenH = 448
+screen = pygame.display.set_mode((screenW,screenH))
+
+
+# I forgot to mention 'Clock'
+# the 'clock' instance help us to control the FPS
+# see below - clock.tick(60) makes the fps to be 60 
+clock = pygame.time.Clock()
+
+# import images & animation here
+bg = pygame.image.load("C:\\Users\\nazri\\Downloads\\Mario\\static_images\\background.png")
+mario = pygame.image.load("C:\\Users\\nazri\\Downloads\\Mario\\static_images\\mario.png")
```

Comment:
Try directly importing static_images/background.png instead of giving the whole path 
You may not be able to do so because you didn't open the whole folder from VS Code (that's the only inconvenience of VS code I think)
To do so, go to File at the top left, click Open Folder, find the folder you want to open (in this case Mario), then you can see your file trees at the left, all the png and folders you have including your py file. In that way you should be able to import directly static_images/xxx.png instead of giving the full path

Reply:
Also, this ensures other people can run your code without needing to change the file path, imaging in the mid of development when we need to important like 30 images, your teammates have to change 30 times too!